src/test/resources/application-test.yml

```diff
@@ -41,5 +41,5 @@ hmpps.sqs:
       arn: arn:aws:sns:eu-west-2:000000000000:11111111-2222-3333-4444-555555555555
 
 feature:
-  async-mode: false
+  async-mode: true
```

Comment:
In tests it will no longer call Prison API to create the Adjudication.  Once SYSCON have done their first sync we'll flip the feature switch to ASYNC in DEV (t3) the